src/map/skill.cpp

```diff
@@ -8484,6 +8485,7 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 					case SC_DONTFORGETME:
 					case SC_FORTUNE:
 					case SC_SERVICE4U:
+					case SC_EP16_2_BUFF_SS:		case SC_EP16_2_BUFF_SC:	case SC_EP16_2_BUFF_AC:
```

Comment:
These need to be moved up to the `continue;` else it will hit the check specific for songs.